deps/curl/lib/vtls/sf_ocsp.c

```diff
@@ -1718,20 +1718,7 @@ OCSP_RESPONSE * getOCSPResponse(X509 *cert, X509 *issuer,
 
 static void printOCSPFailOpenWarning(SF_OTD *ocsp_log, struct Curl_easy *data, bool oob_enable)
 {
-  char *ocsp_log_data = NULL;
-  ocsp_log_data = generateOCSPTelemetryData(ocsp_log);
-  infof(data, "WARNING!!! Using fail-open to connect. Driver is connecting to an "
-              "HTTPS endpoint without OCSP based Certificate Revocation checking "
-              "as it could not obtain a valid OCSP Response to use from the CA OCSP "
-              "responder. Details:%s",ocsp_log_data);
-  if (ocsp_log_data)
-  {
-    if (oob_enable)
-    {
-      sendOOBevent(ocsp_log_data);
-    }
-    if(ocsp_log_data) sf_curl_cJSON_free(ocsp_log_data);
-  }
+  infof(data, "OCSP responder didn't respond correctly. Assuming certificate is not revoked.");
```

Comment:
Can we preserve details as it was before, please?

Reply:
done